blog/index.html

```diff
@@ -62,4 +71,4 @@
          <!-- End right sidebar -->
         </div>
     </div>
-</div>
```

Comment:
I did not remove the div tag,
I don't know why it looks like it has been deleted..